tools/driver/cling.cpp

```diff
@@ -106,6 +107,29 @@ int main( int argc, char **argv ) {
     Interp.loadFile(Lib);
 
   cling::UserInterface Ui(Interp);
+
+  // Get the home directory
+  std::string HomeDir;
+#ifdef __unix__
+  HomeDir = getenv("HOME");
+  HomeDir += "/";
+#elif defined(_WIN32)
+  HomeDir = getenv("USERPROFILE");
+  HomeDir += "\\";
+#endif
+
+  // Look for .clinginit in the home directory and if found process it like a cling script
+  std::string CLingInit = HomeDir + ".clinginit";
```

Comment:
I believe most other tools suffix `_init` which would suggest
```suggestion
  std::string CLingInit = HomeDir + ".cling_init";
```

Reply:
`gdb` uses `.gdbinit` so I stuck with that but `.cling_init` is definitely better.